specifyweb/frontend/js_src/lib/components/WbPlanView/Mapper.tsx

```diff
@@ -385,12 +386,12 @@ export function Mapper(props: {
             {props.isReadOnly ? wbText.dataEditor() : commonText.cancel()}
           </Link.Small>
           {!props.isReadOnly && (
-            <Button.Small
+            <Submit.Save
```

Comment:
Please review **ALL THE PLACES** where you made a change to use a .Save button to check for the following:
1. If you are using Submit where it used Button before, you need to wrap the entire page in a `<Form>`. Alternatively, use `Button.Save` instead.
2. Make sure that after your change did not affect the button size (looks like you went from Button.Small to regular sized button in here)